corehq/apps/registration/views.py

```diff
@@ -117,7 +122,22 @@ def _create_new_account(self, reg_form):
             web_user = WebUser.get_by_username(new_user.username)
             web_user.phone_numbers.append(reg_form.cleaned_data['phone_number'])
             web_user.save()
-        track_workflow(new_user.email, "Requested new account")
+
+        is_mobile = reg_form.cleaned_data.get('is_mobile')
+        email = reg_form.cleaned_data['email']
+
+        if is_mobile:
+            toggles.MOBILE_SIGNUP_REDIRECT_AB_TEST_CONTROLLER.set(email, True)
+        track_workflow(new_user.email,
+                       "Requested new account",
+                       {
+                           'mobile_visitor': reg_form.cleaned_data.get('is_mobile'),
```

Comment:
Nitpick: might name these properties something more distinct, like the `new_phone_jan2016`-type strings used in ab_tests.py, or something else to make it obvious that this is temporary and a test. I suspect this data will live forever in kiss.

Reply:
Yikes. Updated